src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/DiagController.cs

```diff
@@ -48,9 +47,19 @@ public Task<ActionResult> GetDump(int? pid, [FromQuery]DumpType type = DumpType.
             {
                 OperationResult<Stream> result = await _diagnosticServices.GetDump(pid, type);
 
-                //Compression is done automatically by the response
-                //Chunking is done because the result has no content-length
-                return CreateFileStreamResult(result, $"coredump_{result.Pid}{DumpFileExtension}");
+                FormattableString dumpFileName;
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
My original comment was vague, but I really meant "not coredump". This is much better though :-). One other suggestion that has been floated before (that has the same restrictions in a container setting) was to also return the process name along the pid in the processes command.

Reply:
Yeah, I can easily add that in right now since this only works with processes in the same process space at this time. When we add the reverse diagnostic server, we might not be able to provide that information too easily.